tests/unit/applications/test_models.py

```diff
@@ -1,7 +1,31 @@
+from django.urls import reverse
+
 from ...factories import ResearcherRegistrationFactory
 
 
-def test_job_str():
+def test_researcherregistration_get_delete_url():
+    researcher = ResearcherRegistrationFactory()
+
+    url = researcher.get_delete_url()
+
+    assert url == reverse(
+        "applications:researcher-delete",
+        kwargs={"pk": researcher.application.pk, "researcher_pk": researcher.pk},
+    )
+
+
+def test_researcherregistration_get_edit_url():
+    researcher = ResearcherRegistrationFactory()
+
+    url = researcher.get_edit_url()
+
+    assert url == reverse(
+        "applications:researcher-edit",
+        kwargs={"pk": researcher.application.pk, "researcher_pk": researcher.pk},
+    )
```

Comment:
I'm not sure what the point of this kind of test is -- the test is exactly duplicating the code.  If we remove the tests and the coverage drops, that indicates either that these methods are never used, or that the code that uses the methods is not tested.